src/app/compressed-air-assessment/compressor-summary-table/compressor-summary-table.component.html

```diff
@@ -0,0 +1,69 @@
+<div class="d-flex flex-column">
+    <div class="d-flex justify-content-center">
+        <div class="pt-2 pr-1 group-label">
+            <span *ngIf="inReport">Baseline Compressor Summary</span>
+            <span *ngIf="!inReport">Compressor Summary</span>
+        </div>
+    </div>
+    <div class="pl-2 pt-3 pr-2">
+        <table #profileTable class="table table-hover bg-white">
+            <thead>
+                <tr>
+                    <th></th>
+                    <th>Day Type</th>
+                    <th class="bold" *ngFor="let compressor of compressorInventoryItems">
+                        {{compressor.name}}
+                    </th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <th [attr.rowspan]="dayTypes.length + 1">
```

Comment:
It looks like you can use compressorSummaries length here and just keep one source of truth/dimensions

Reply:
the row span needs to be `dayTypes.length + 1` because the number of day types determines how many rows that cell needs to span.